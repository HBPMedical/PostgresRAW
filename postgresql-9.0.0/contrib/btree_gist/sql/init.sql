--
-- first, define the datatype.  Turn off echoing so that expected file
-- does not depend on contents of btree_gist.sql.
--
SET client_min_messages = warning;
\set ECHO none
\i btree_gist.sql
\set ECHO all
RESET client_min_messages;
