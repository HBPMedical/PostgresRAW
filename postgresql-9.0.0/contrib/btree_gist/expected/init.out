--
-- first, define the datatype.  Turn off echoing so that expected file
-- does not depend on contents of btree_gist.sql.
--
SET client_min_messages = warning;
\set ECHO none
RESET client_min_messages;
